.circleci/config.yml

```diff
@@ -156,6 +156,7 @@ jobs:
           command: |
             for FILE in $(ls docker)
             do
+              if [[FILE == "test.sh"]]; then continue
```

Comment:
I think you'll also need to ignore the `tests` directory to solve the second error:

```
Error response from daemon: manifest for hyperledger/besu:develop-tests not found
```

Reply:
ok yep and about 99 other problems with my bash. hopefully fixed now!